docs/src/main/sphinx/object-storage/metastores.md

```diff
@@ -80,6 +80,14 @@ are also available. They are discussed later in this topic.
 * - `hive.metastore-refresh-max-threads`
   - Maximum threads used to refresh cached metastore data.
   - `10`
+* - `hive.user-metastore-cache-ttl`
+  - Duration of how long cached metastore statistics which are user specific
+    in user impersonation scenarios are considered valid.
+  - `10`
+* - `hive.user-metastore-cache-maximum-size`
+  - Maximum number of metastore data objects in the Hive metastore cache
+    which are user specific in user impersonation scenarios .
+  - `10`
```

Comment:
Could you change `10` to `1000`? 
https://github.com/trinodb/trino/blob/b519bdf7d90aeb414121788999b48930cef6d934/plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/cache/ImpersonationCachingConfig.java#L26